### DISCUSSION TITLE
pypi mirrors support?

### DISCUSSION THREAD

**Discussion by ormeilu** [astral-sh/rye#902] [Q&A] [answered]
Does `rye` support setting custom pypi mirrors repository or custom repository at all?
Like, for example - https://www.sonatype.com/products/sonatype-nexus-repository

Or `rye` inherits pip setting from `/etc/pip.conf` and if I add mirror there - `rye` will also have that mirror?

**Comment by ischaojie** [comment, marked as answer]
You means https://rye-up.com/guide/sources/ ?

**Reply by ormeilu** [reply]
Yep! Thx
